### ISSUE TITLE
Getting [warn] Using execute/3 with a single

### ISSUE THREAD

**Issue by konstantinzolotarev** [mmmries/gnat#85]
Getting this message every time calling any `pub/sub` function.
```
[warn] Using execute/3 with a single event value is deprecated. Use a measurement map instead.
```

**Comment by mmmries** [comment]
Any chance you can drop your `mix.lock` file here? I'm thinking this warning is coming from one of the dependencies and wondering which versions you are using.

**Comment by konstantinzolotarev** [comment]
```
%{
  "artificery": {:hex, :artificery, "0.4.0", "e0b8d3eb9dfe8f42c08a620f90a2aa9cef5dba9fcdfcecad5c2be451df159a77", [:mix], [], "hexpm"},
  "distillery": {:hex, :distillery, "2.0.12", "6e78fe042df82610ac3fa50bd7d2d8190ad287d120d3cd1682d83a44e8b34dfb", [:mix], [{:artificery, "~> 0.2", [hex: :artificery, repo: "hexpm", optional: false]}], "hexpm"},
  "gen_stage": {:hex, :gen_stage, "0.14.1", "9d46723fda072d4f4bb31a102560013f7960f5d80ea44dcb96fd6304ed61e7a4", [:mix], [], "hexpm"},
  "gnat": {:hex, :gnat, "0.6.0", "4d5e77e726a77cc29a35793782d62b4aad030f21be0045b44eddd4cd9413eba3", [:mix], [{:jason, "~> 1.1", [hex: :jason, repo: "hexpm", optional: false]}, {:nimble_parsec, "~> 0.5", [hex: :nimble_parsec, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.3", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm"},
  "jason": {:hex, :jason, "1.1.2", "b03dedea67a99223a2eaf9f1264ce37154564de899fd3d8b9a21b1a6fd64afe7", [:mix], [{:decimal, "~> 1.0", [hex: :decimal, repo: "hexpm", optional: true]}], "hexpm"},
  "nimble_parsec": {:hex, :nimble_parsec, "0.5.0", "90e2eca3d0266e5c53f8fbe0079694740b9c91b6747f2b7e3c5d21966bba8300", [:mix], [], "hexpm"},
  "porcelain": {:hex, :porcelain, "2.0.3", "2d77b17d1f21fed875b8c5ecba72a01533db2013bd2e5e62c6d286c029150fdc", [:mix], [], "hexpm"},
  "telemetry": {:hex, :telemetry, "0.4.0", "8339bee3fa8b91cb84d14c2935f8ecf399ccd87301ad6da6b71c09553834b2ab", [:rebar3], [], "hexpm"},
}
```

**Comment by konstantinzolotarev** [comment]
This warning is from `:telemetry`

**Comment by mmmries** [comment]
There is a PR open to update to the latest telemetry https://github.com/mmmries/gnat/pull/86

**Comment by mmmries** [comment]
The fix for this has been released as `0.6.1`. It will remove all warnings from telemetry.

**Comment by mmmries** [comment]
Thank you again for reporting this and for following up with additional details.
